### PR TITLE
[keycloak] Apply secret type configured in values file

### PR DESCRIPTION
**Pull request by WaylandWong**
specify secrets's type.
https://github.com/codecentric/helm-charts/issues/324

### CODE REVIEW

File: charts/keycloak/templates/secrets.yaml [hunk 1]
```diff
@@ -14,7 +14,11 @@ metadata:
   {{- range $key, $value := $values.labels }}
   {{- printf "%s: %s" $key (tpl $value $ | quote) | nindent 4 }}
   {{- end }}
+{{- if $values.type }}
+type: {{ $values.type }}
+{{- else }}
 type: Opaque
+{{- end }}
```
Comment (unguiculus) [inline charts/keycloak/templates/secrets.yaml:21]: I'd suggest you simply this:
```suggestion
type: {{ default $values.type "Opaque" }}
```

**Review by unguiculus — CHANGES_REQUESTED** [review]
Thanks. Please fix DCO and bump the chart version.

Also, please update the comment in the values file:
https://github.com/codecentric/helm-charts/blob/master/charts/keycloak/values.yaml#L209-L214

File: charts/keycloak/templates/secrets.yaml (hunk 1 above)
Comment (WaylandWong) [reply charts/keycloak/templates/secrets.yaml:21]: Good idea. And I think it would be 
```
type: {{ default "Opaque" $values.type}}
```

**Comment by WaylandWong** [comment]
I change the version to `9.9.5`, so please merge #40 first.

File: charts/keycloak/templates/secrets.yaml (hunk 1 above)
Comment (unguiculus) [reply charts/keycloak/templates/secrets.yaml]: You're right, of course 😉

**Review by unguiculus — APPROVED** [review]

**Pull request merged**